src/components/Settings/SettingsMain.tsx

```diff
@@ -41,6 +41,8 @@ const messages = defineMessages({
   validationApplicationTitle: 'You must provide an application title',
   validationApplicationUrl: 'You must provide a valid URL',
   validationApplicationUrlTrailingSlash: 'URL must not end in a trailing slash',
+  quotaPeriod: 'Default Quota Period (days)',
+  quotaQuantity: 'Default Quota Quantity (requests)',
```

Comment:
```suggestion
  quotaQuantity: 'Default Request Limit',
```

I think that while "Quota" makes sense for us as devs, "Request Limit" would be clearer and less confusing for the end user.  We should also avoid parentheses in labels / field names; if parentheses are necessary, it probably needs to be reworded/renamed or have a "tip"/description added.

Also, I think that this should appear as one setting/option/row to the end user even though it's really two values, like:
```
Default Request Limit            [      ] requests per [      ] days
```

We also need `Yup` validation on these fields ("You must enter a number of requests" / "You must enter a number of days").

Reply:
On second thought, the `quotaPeriod` setting should probably be a dropdown with options for day/week/month (we can still store the value as 1/7/30 days).

```
Default Request Limit            [      ] requests per [ day   v ]
                                                       [ week    ]
                                                       [ month   ]
```

Then, we don't need validation on that field, and the `quotaQuantity` setting can be optional (so quotas only apply if a value is set).